webserver/views/api/v1/core.py

```diff
@@ -10,12 +10,14 @@
 from db.data import submit_low_level_data, count_lowlevel
 from db.exceptions import NoDataFoundException, BadDataException
 from webserver.decorators import crossdomain
+from brainzutils.ratelimit import ratelimit
 
 bp_core = Blueprint('api_v1_core', __name__)
 
 
 @bp_core.route("/<uuid:mbid>/count", methods=["GET"])
 @crossdomain()
+@ratelimit()
```

Comment:
I'd like to see at least one test to see that we're returning ratelimiting headers on one of these methods